lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -275,6 +275,8 @@ describe('ReportUIFeatures', () => {
     let dropDown;
 
     beforeEach(() => {
+      // render is a prerequisite for initFeatures
+      render(sampleResults);
```

Comment:
WDYT about moving this up as the last step in the global `beforeAll`? I think then it becomes clear what its purpose is and can avoid the comment too :)

Reply:
SGTM, thanks for the review!